cfdm/mixin/netcdf.py

```diff
@@ -272,7 +499,7 @@ def nc_get_dimension(self, default=ValueError()):
             `str`
                 The netCDF dimension name.
 
-        **Examples:**
+        **Examples**
```

Comment:
Maybe we should do a `sed` style replacement on all cases of 'Examples:' to remove the colon everywhere in a single commit/PR, instead of doing it as we see it. I can do that quickly, if you like (since I have the right command at my disposal :grin:).

Reply:
Great - please go ahead with the sensible way!

Reply:
Sure, though I'll wait until this is merged to do it to avoid potential merge conflicts arising if other code nearby has changed.